### DISCUSSION TITLE
Validation

### DISCUSSION THREAD

**Discussion by DarkRaptor8** [Crownicles/Crownicles#1687] [Suggestions fermées]
![Screenshot_20221227_162418_Discord](https://user-images.githubusercontent.com/96015334/209868310-f0f085fd-1216-4b70-9cda-2449e26a30ed.jpg)
Devoir valider une deuxième fois pour éviter toute les erreurs comme ça et je sais que je ne suis pas le seul

**Comment by BastLast** [comment]
cette suggestion a déjà été proposée plusieurs fois, nous avons fait le choix de laisser une unique confirmation car il serait très lourd pour les joueurs qui savent ce qu'ils font et pour les performances de demander plusieurs fois l'avis des gens. d'autant que plusieurs personnes pourraient ne pas remarquer ou ne pas comprendre le second menu.

Pour éviter les missclicks, il est possible de juste attendre au lieu de répondre au menu et ainsi d'avoir le choix :x: effectué automatiquement.

Je déplace dans les idées refusées
